### PR TITLE
RavenDB-19170 Adjust Lucene methods for memoization in VoronInputIndex

### PR DESCRIPTION
**Pull request by maciejaszyk**
https://issues.hibernatingrhinos.com/issue/RavenDB-19170/Memoization-in-VoronIndexInput

### CODE REVIEW

File: src/Lucene.Net/Store/BufferedIndexInput.cs [hunk 1]
```diff
@@ -168,7 +168,7 @@ public override void  ReadBytes(byte[] b, int offset, int len, bool useBuffer, I
 			}
 		}
 		
-		private void  Refill(IState state)
+		public void  Refill(IState state)
```
Comment (arekpalinski) [inline src/Lucene.Net/Store/BufferedIndexInput.cs]: Does it need to be public? Wouldn't `protected` be sufficient?

File: src/Directory.Build.props [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <Project>
   <PropertyGroup>
-    <Version>3.0.54000</Version>
-    <FileVersion>3.0.54000.0</FileVersion>
+    <Version>3.0.54100</Version>
+    <FileVersion>3.0.54100.0</FileVersion>
```
Comment (arekpalinski) [inline src/Directory.Build.props]: Let's set it to `3.0.54001`

Comment (maciejaszyk) [reply src/Directory.Build.props]: done

File: src/Lucene.Net/Store/BufferedIndexInput.cs (hunk 1 above)
Comment (maciejaszyk) [reply src/Lucene.Net/Store/BufferedIndexInput.cs]: Yeah, done

**Pull request merged**